src/__tests__/fixtures/regression.css

```diff
@@ -0,0 +1,7 @@
+@custom-selector :--button .button;
+
+:--button {
+  &--active {
```

Comment:
@MoOx just remove `--wat` from there and it will fail as well because the issue is `nesting` + `custom selector` is not even about use that notation (`&--something`)